modules/imgproc/src/opencl/resize.cl

```diff
@@ -220,6 +220,78 @@ __kernel void resizeLN(__global const uchar * srcptr, int src_step, int src_offs
     }
 }
 
+#elif defined INTER_LINEAR_EXACT
+
+__kernel void precomputeCoeffs(__read_only image2d_t src, __write_only image2d_t coeffsTex, float inv_fx, float inv_fy) {
```

Comment:
For some reason OpenCV tries to use regular buffers in OpenCL implementations instead of Image2D objects (maybe @opencv-alalek knows better).

Reply:
For good reason. That was just one of the experiments (read above) but image2d_t needs support for the corresponding image formats.